src/usb.c

```diff
@@ -1252,6 +1259,7 @@ handle_acquire_devices (XdpDbusUsb            *object,
 static gboolean
 handle_finish_acquire_devices (XdpDbusUsb            *object,
                                GDBusMethodInvocation *invocation,
+                               const char            *object_path,
```

Comment:
Do we have a guarantee that this isn't the object path of another client?

Reply:
I have added a check so that if it doesn't know about the object_path / token it will return an error.